build/config/phone/custom-prefs.js

```diff
@@ -2,3 +2,9 @@
 user_pref('devtools.responsiveUI.customWidth', 320);
 user_pref('devtools.responsiveUI.customHeight', 480);
 user_pref('devtools.responsiveUI.currentPreset', 'custom');
+
+user_pref('dom.caches.enabled', true);
+user_pref('dom.serviceWorkers.enabled', true);
+user_pref('dom.webcomponents.enabled', true);
+user_pref('dom.apps.developer_mode', true);
+user_pref('network.disable.ipc.security', true);
```

Comment:
Perhaps we can put a comment specifying the bug that will fix this (I cannot remember right now xD)